Services/Music_Portal.Services.Interfaces/ILastFmService.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
-using Interfaces.Models;
+using Services.Interfaces.Models;
 
-namespace Interfaces
+namespace Services.Interfaces
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Services.Interfaces`